Dockerfile

```diff
@@ -50,6 +50,7 @@ RUN apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv E1DD270288B4E60
  && rm -rf /var/lib/apt/lists/*
 
 COPY assets/build/ ${GITLAB_BUILD_DIR}/
+ARG GITLAB_EDITION=ce
```

Comment:
Since GitLab moved to a single codebase (see https://about.gitlab.com/blog/2019/08/23/a-single-codebase-for-gitlab-community-and-enterprise-edition/) the naming has changed (although the redirects still work!). It's not `gitlab-org/gitlab-foss` (suffix is `-foss`; former `-ce`) and `gitlab-org/gitlab` (no suffix; former `-ee`).

FYI removing the EE folder from `gitlab-org/gitlab` (`rm -fr ./ee`) gives you basically the FOSS version.

Reply:
Correct, both EE => CE and CE => EE procedures require some manual work:
1. https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/downgrade_ee_to_ce/README.md
2. https://gitlab.com/gitlab-org/gitlab/-/blob/master/doc/update/upgrading_from_ce_to_ee.md

Some or procedures are not applicable to docker environments as they are applied at container start or build (cache clearing, deps installation, db migrations). So migrations are also the discussion point for test and documenting using sameersbn docker images.

I mean now conflict resolving is mainly use the new url for CE, as it made in master at 7ca29dfce0cedf51ff47da24c813c9b9984dddbc